src/public/app/widgets/tab_row.js

```diff
@@ -187,6 +187,14 @@ const TAB_ROW_TPL = `
         cursor: pointer;
     }
     
+    .tab-row-widget .note-tab:hover .note-tab-wrapper {
+        background-color: var(--accented-background-color);
+    }
+    
+    .tab-row-widget .note-tab[active]:hover .note-tab-wrapper {
+        background-color: var(--more-accented-background-color);
+    }
+    
```

Comment:
This has changed

Reply:
Yes, I can see the code change, but no visual change. It literally looks exactly the same.

Reply:
Oh I see, it's only in light mode (I guess because the css variables are probably the same). In dark mode it looks like this (maybe we should update the light mode variables?):

https://user-images.githubusercontent.com/50424412/144488765-eee8d28f-a7cf-493e-8c8f-2f7734be8079.mp4

Reply:
I see. Tab coloring is controlled by separate CSS variables (e.g. `--active-tab-background-color`) so I guess better way would be to define new "hover background" variable for tab which would solve this theme inconsistency.

Reply:
Don't forget the light theme also please.